src/vm/docs/mod.rs

```diff
@@ -85,10 +85,10 @@ contract principal.",
 };
 
 const TOTAL_LIQUID_USTX_KEYWORD: KeywordAPI = KeywordAPI {
-    name: "total-liquid-ustx",
+    name: "stx-liquid-supply",
```

Comment:
I had `ustx` because the value is in _micro_ STX.

Reply:
Yeah I think I've also raised this in the past.
All the other clarity **stx** functions are dealing with micro STX (stx-get-balance, stx-transfer).
We should either use the **ustx** prefix in the other functions, or stick with **stx**, but a mix of the two is very confusing.

Reply:
Fair point :+1: